### PR TITLE
Mobile 1043

### PR DESCRIPTION
**Pull request by rlepinski**
### What do these changes do?
Updates Android SDK to 12.1.

### Why are these changes necessary?
We need a modern version of our SDK to support new features.

### How did you verify these changes?
Ran sample app, verified message center, named user, and tags work.

### CODE REVIEW

**Comment by rlepinski** [comment]
CI is failing because of the cordova pod issue

File: README.md [hunk 1]
```diff
@@ -41,6 +41,17 @@ You would only run `pod repo update` if you have the specs-repo already cloned o
 #### Android
  - Android [FCM Setup](https://docs.airship.com/platform/android/getting-started/#fcm-setup)
 
+##### Jetpack / AndroidX
+
+This plugin requires modern Jetpack libraries (AndroidX). If the application includes plugins that are still on the old Android
+Support libraries, you will face build issues. The quickest way to work around the issue is to install `cordova-plugin-androidx`
+and `cordova-plugin-androidx-adapter` that will automatically enable Jetpack and migrate plugins in the application:
```
Comment (marc-scig) [inline README.md]: , which will automatically enable...

File: scripts/create_sample.sh [hunk 2]
```diff
@@ -44,15 +44,17 @@ cp $ROOT_PATH/Example/index.html www/index.html
 cp $ROOT_PATH/Example/css/* www/css
 cp $ROOT_PATH/Example/js/* www/js
 
-# add the device plugin
+# add required plugins
 cordova plugin add cordova-plugin-device
+cordova plugin add cordova-plugin-androidx-adapter
```
Comment (marc-scig) [inline scripts/create_sample.sh]: if these are optional based on the app's needs is there a way we can communicate that?

Comment (rlepinski) [reply scripts/create_sample.sh]: I put it in the README and we will put it in the docs. I am not sure if we can force people to see if when installing the plugin.

**Pull request merged**